# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/mysql/asm_bitops_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./client/mysqltest.c	2005-05-01 08:53:22.012954280 +0200
+++ ./client/mysqltest.c	2005-05-01 08:55:04.275408024 +0200
@@ -3901,6 +3901,9 @@
   uint len;
 } FOLLOWS;
 
+// avoid conflict with asm/bitops.h
+#define set_bit my_set_bit
+#define clear_bit my_clear_bit
 
 static int init_sets(REP_SETS *sets,uint states);
 static REP_SET *make_new_set(REP_SETS *sets);
